frontstage/views/passwords/forgot_password.py

```diff
@@ -51,19 +45,13 @@ def post_forgot_password():
             if exc.status_code == 404:
                 logger.error('Requesting password change for email registered'
                              ' on OAuth2 server but not in party service')
-                template_data = {"error": {"type": {"Email address is not registered"}}}
-                return render_template('passwords/forgot-password.html', form=form, data=template_data)
+                return render_template('passwords/forgot-password.html', form=form)
```

Comment:
Should this also go to `passwords/forgot-password.check-email.html` as above? Seems strange to do it for one and then present no information to the user when Party check fails.